README.md

```diff
@@ -1,224 +1,266 @@
+You can test posting directly to the server locally with:
 ```
-curl -i -H "Authorization: Bearer token" -F "image=@/path/to/image" -F "caption=caption" -F "location=location" https://{host}/posts
+[local]$ ./scripts/post_image_local.sh {host} /path/to/image.jpg "caption" "location"
+[local]$ ./scripts/post_text_local.sh {host} /path/to/text.md "location"
 ```
 
+**Note:** Individual post pages don't work locally because they rely on nginx to
+reroute their paths to the `.html` files.
+
 ### Icons
-Fill in the `app/sites/{host}/content/icons` directory with the following files so your site will automatically serve a favicon and apple touch icon:
+
+Fill in the `app/sites/{host}/content/icons` directory with the following files
+so your site will automatically serve a favicon and Apple touch icon:
+
 ```
 favicon.png (64x64)
 apple-touch-icon.png (180x180)
 ```
 
-Then push your site to your remote server:
-```
-./scripts/push.sh {host}
-```
-
 ### Regenerating
-If you ever edit anything in `app/sites/{host}/data`, you can regenerate your site with:
+
+If you ever edit anything in `app/sites/{host}/data`, you can regenerate your
+site with:
 ```
-python app/content_manager.py regenerate -s {host}
+[local]$ python app/content_manager.py {host} generate
 ```
 
 ### Multiple Sites
-Managing multiple sites on the same machine is pretty simple (you can host them on different servers if you prefer).
+
+Managing multiple sites on the same machine is pretty simple (you can host them
+on different servers if you prefer).
 
 **1. Create a new site on your server**
+
 ```
-python app/site_manager.py create -s "{new_host}" -t "new site title" -d "new site description"
+[server]$ python app/site_manager.py create -s "{new_host}" -t "new site title" -d "new site description" -f "date format"
 ```
 
 **2. Encrypt the traffic**
+
 ```
-sudo certbot --nginx -d {new_host} -d www.{new_host}
+[server]$ sudo certbot --nginx -d {new_host} -d www.{new_host}
 ```
 
 **3. Update and restart nginx**
+
 ```
-sudo python3 app/site_manager.py update_nginx
-sudo systemctl restart nginx
+[server]$ sudo python3 app/site_manager.py update_nginx
+[server]$ sudo systemctl restart nginx
 ```
 
-## Caveats
-The site design is simple and inflexible, but there are already plenty of tools for managing complex sites. I wanted to post photos to an https site from my laptop and phone, and I wanted to serve them as static content.
+Caveats
+-------
+
+The site design is simple and inflexible, but there are already plenty of tools
+for managing complex sites. I wanted to post photos to an https site from my
+laptop and phone, and I wanted to serve them as static content.
 
-That said, feel free to fork the project and mess around with the `template` directory - there are css files in `template/content/styles`. Restructuring the HTML requires code changes in `app/post.py`.
+That said, *feel free to fork* the project and mess around with the `template`
```

Comment:
Point taken, maybe italics is a little ragey for the general public

Reply:
You're a little ragey for the general public